pages/docs/opentelemetry/get-started.mdx

```diff
@@ -238,23 +238,41 @@ In addition, we map many GenAI specific properties to properties in the Langfuse
 First and foremost, we stick to the [OpenTelemetry Gen AI Conventions](https://opentelemetry.io/docs/specs/semconv/attributes-registry/gen-ai/), but also map vendor specific properties from common frameworks.
 All attributes and resourceAttributes are available within the Langfuse `metadata` property as a fallback.
 
-Below, we share a non-exhaustive list of mappings that Langfuse applies:
+Langfuse uses the `langfuse.*` namespace to map properties from OpenTelemetry to the Langfuse data model.
+This enables features like linking [Prompts](/docs/prompts/get-started) to spans, adding [tags](/docs/tracing-features/tags), or marking traces as public.
+Attributes in the Langfuse namespace take precedence over the generic mapping we specify below.
+Some of the attributes are only taken into consideration if they are set on the root span. We mark those in the table below.
+The following Langfuse specific properties are supported:
+
+| OpenTelemetry Attribute   | Langfuse Property | Type       | Root Span only | Description                                                                    |
+|---------------------------|-------------------|------------|----------------|--------------------------------------------------------------------------------|
+| `langfuse.session.id`     | `sessionId`       | `string`   | `true`         | The [session ID](/docs/tracing-features/sessions) for the request.             |
+| `langfuse.user.id`        | `userId`          | `string`   | `true`         | The [user ID](/docs/tracing-features/users) for the request.                   |
+| `langfuse.public`         | `public`          | `boolean`  | `true`         | Mark the trace as public. This can be changed later in the Langfuse UI.        |
```

Comment:
```suggestion
| `langfuse.public`         | `public`          | `boolean`  | `true`         | Mark the trace as public to be able to [share it via url](/docs/tracing-features/url). This can be changed later in the Langfuse UI.        |
```